modules/sentry/src/sentry.test.js

```diff
@@ -0,0 +1,147 @@
+const Raven = require("raven-js");
+const Sentry = require("./sentry").default;
+
+jest.mock("../index.js");
+jest.mock("../__mocks__/raven-js.js");
```

Comment:
You must call `jest.mock` **before** importing the modules, it will not works else.

else to mock a lib you have to call the function like this: `jest.mock("raven-js")`, and the mock file must be in a `__mock__` folder beside the `node_modules` folder, please read the JEST documentation for more info → https://jestjs.io/docs/en/bypassing-module-mocks